R/new.R

```diff
@@ -13,6 +13,15 @@ generator_funs$new <- function(...) {
     if (!identical(portable, inherit$portable))
       stop("Sub and superclass must both be portable or non-portable.")
 
+    if (!identical(cloneable, inherit$cloneable)) {
+      message(c(
```

Comment:
I think this should be a message and not a warning, since, in some contexts, such a mismatch is unavoidable.